foreign_cc/built_tools/make_build.bzl

```diff
@@ -10,6 +10,7 @@ load("//foreign_cc/private:shell_script_helper.bzl", "os_name")
 
 def _make_tool_impl(ctx):
     script = [
+        "##script_prelude##",
```

Comment:
Doesn't the prelude get added in the built tools framework?

Reply:
Yes you're right, this was my original change but you've added the framework bits since then so I should remove this.